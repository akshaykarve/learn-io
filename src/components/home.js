import React from 'react';

const HomeController = () =>{
    return (
    <div>
        <p>Home!</p>
    </div>
    )
}
export default HomeController;