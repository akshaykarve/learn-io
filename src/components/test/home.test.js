import React from 'react';
import renderer from 'react-test-renderer';
import {default as HomeController} from "../home";
describe("Home Controller", ()=>{
	it('home test', () => {
		
		const tree = renderer.create(
			<HomeController isSignedIn={false}
			/>
		).toJSON();
		expect(tree).toMatchSnapshot();
	});

});