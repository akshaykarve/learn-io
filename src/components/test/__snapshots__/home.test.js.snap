// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home  home 1`] = `
<div
  className="appStyle"
>
  <div
    className="pa"
  >
    <input
      className="searchBoxStyle"
      onChange={[Function]}
      placeholder="Search platform"
      type="search"
    />
  </div>
  <div
    className="dropdown"
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="dropdown-toggle btn btn-success"
      disabled={false}
      id="dropdown-basic"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#cdecff",
          "color": "#000",
        }
      }
      type="button"
    >
      Platfroms per page: 
      10
    </button>
  </div>
  <div />
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Previous
  </button>
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Next
  </button>
</div>
`;

exports[`Home  home test 1`] = `
<div
  className="appStyle"
>
  <div
    className="pa"
  >
    <input
      className="searchBoxStyle"
      onChange={[Function]}
      placeholder="Search platform"
      type="search"
    />
  </div>
  <div
    className="dropdown"
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="dropdown-toggle btn btn-success"
      disabled={false}
      id="dropdown-basic"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#cdecff",
          "color": "#000",
        }
      }
      type="button"
    >
      Platfroms per page: 
      10
    </button>
  </div>
  <div />
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Previous
  </button>
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Next
  </button>
</div>
`;

exports[`Home Controller home test 1`] = `
<div
  className="appStyle"
>
  <div
    className="pa"
  >
    <input
      className="searchBoxStyle"
      onChange={[Function]}
      placeholder="Search platform"
      type="search"
    />
  </div>
  <div
    className="dropdown"
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="dropdown-toggle btn btn-success"
      disabled={false}
      id="dropdown-basic"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#cdecff",
          "color": "#000",
        }
      }
      type="button"
    >
      Platfroms per page: 
      10
    </button>
  </div>
  <div />
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Previous
  </button>
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Next
  </button>
</div>
`;

exports[`Home Controller home test 2`] = `
<div
  className="appStyle"
>
  <div
    className="pa"
  >
    <input
      className="searchBoxStyle"
      onChange={[Function]}
      placeholder="Search platform"
      type="search"
    />
  </div>
  <div
    className="dropdown"
  >
    <button
      aria-expanded={false}
      aria-haspopup={true}
      className="dropdown-toggle btn btn-success"
      disabled={false}
      id="dropdown-basic"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#cdecff",
          "color": "#000",
        }
      }
      type="button"
    >
      Platfroms per page: 
      10
    </button>
  </div>
  <div />
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Previous
  </button>
  <button
    className="homeButton"
    disabled={true}
    style={
      Object {
        "color": "grey",
      }
    }
  >
    Next
  </button>
</div>
`;
